### PR TITLE
Flyswatter changes 2: Electric Boogaloo

### PR DESCRIPTION
**Pull request by Thelolswat**
:cl: TheLoLSwat
tweak: We at Nanotrasen have learned of how weak your flyswatters are, and we are outraged. We swapped your weak-sauce swatters with stronger ones, that are bound to make some new noises
/:cl:

[why]: flyswatters have been weak for too long :(

### CODE REVIEW

File: hippiestation/code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -76,4 +76,9 @@
 
 /obj/item/staff/Initialize()
 	. = ..()
-	AddComponent(/datum/component/spell_catalyst)
\ No newline at end of file
+	AddComponent(/datum/component/spell_catalyst)
+
+/obj/item/melee/flyswatter
+	force = 2
+	throwforce = 2
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/objects/items/weaponry.dm]: Overpowered.

**Review by Manly-Dwarf — APPROVED** [review]

File: hippiestation/code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (Thelolswat) [reply hippiestation/code/game/objects/items/weaponry.dm]: Yes

Comment (ghost) [reply hippiestation/code/game/objects/items/weaponry.dm]: disgusting

**Review by JohnGinnane — DISMISSED** [review]
do we need head input?

**Comment by ghost** [comment]
mergae?

**Comment by Thelolswat** [comment]
For too long those filthy insects have had too much power on our wonderful station

**Comment by ghost** [comment]
why not +1

**Comment by dreamcrab** [comment]
nooooooooooooo

**Review by Carbonhell — DISMISSED** [review]

**Comment by Carbonhell** [comment]
i cant understand if stab is sarcastic or not
Waiting for him

**Comment by JohnGinnane** [comment]
@dreamcrab pls respond)

**Comment by Pyko1** [comment]
not against, but don't really know why either.

Also, merge conflicts.

**Review by JohnGinnane — APPROVED** [review]
Code looks good now

**Comment by Carbonhell** [comment]
aight merging

**Pull request merged**